### PR TITLE
Feature/wgrnd 626

### PR DESCRIPTION
**Pull request by VladislavAslanov482**
## Type of change
Please delete options that are not relevant.
- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
# Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/components/containers/Profile/index.test.js [hunk 1]
```diff
@@ -11,3 +11,11 @@ it('Render without crashing', () => {
   expect(wrapper.find('.profileImage')).to.have.lengthOf(1);
   expect(wrapper.find('.userName')).to.have.lengthOf(1);
 });
+
+it('Calls functions Search', () => {
```
Comment (boivlad) [inline src/components/containers/Profile/index.test.js:15]: Нету смысла вручную вызывать метод componentDidMount, так как при рендере он и так вызывается + не нужно в пропсы передавать метод componentDidMount

**Comment by VladislavAslanov482** [comment]
> Где тесты на новый функционал?

Добавил

File: src/components/containers/Buttons/index.test.js [hunk 2]
```diff
@@ -17,12 +17,18 @@ it('Renders components Buttons', () => {
...
 
 it('Renders components Buttons', () => {
   const wrapper = shallow(<Buttons
     uploadFile={() => {
     }}
+    goHome={() => {
+    }}
   />);
   wrapper.instance().beforeUpload();
+  wrapper.instance().goHome();
```
Comment (boivlad) [inline src/components/containers/Buttons/index.test.js:33]: Для этого нужно создавать новый кейс тестирования, а не всё вместе

Comment (VladislavAslanov482) [reply src/components/containers/Buttons/index.test.js:33]: Исправил

**Pull request merged**